app/frontend/src/locales/da/translation.json

```diff
@@ -59,7 +59,9 @@
         "showSupportingContent": "Vis understøttende indhold",
         "speakAnswer": "Afspil svar",
         "info": "Info",
-        "save": "Gem"
+        "save": "Gem",
+        "copy": "Kopier svar",
```

Comment:
@EMjetrot Can you review these two strings, "Kopier svar" for "Copy", and "Kopieret!" for copied?

Reply:
In that case, we'd need to apply this change across all translations for consistency. Personally, I find "Copy answer" a bit more descriptive than just "Copy", but I understand the simplicity of the shorter version as well.

Reply:
So I checked chatgpt.com as they've got a similar copy button and they only use "Copy", so that's an argument for "Copy".

I think we can safely shorten the romance/european languages based on our own knowledge and what chatgpt.com is using, and I've asked @bnodir to comment about the Japense change since that's where I feel the least confident.